Framework/DataObjects/src/EventList.cpp

```diff
@@ -1567,6 +1632,93 @@ void EventList::compressEventsParallelHelper(
     out.insert(out.end(), outputs[thread].begin(), outputs[thread].end());
 }
 
+template <class T>
+inline void EventList::compressFatEventsHelper(
```

Comment:
Will some compiler really inline this?

Reply:
I have no idea. It was the way `CompressEventsHelper` was declared. Rather than asking questions I just copied the signature.